internal/provider/resource_password.go

```diff
@@ -121,19 +121,21 @@ func (r *passwordResource) Delete(ctx context.Context, req resource.DeleteReques
 func (r *passwordResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
 	id := req.ID
 
-	state := passwordModelV2{
-		ID:         types.String{Value: "none"},
-		Result:     types.String{Value: id},
-		Length:     types.Int64{Value: int64(len(id))},
-		Special:    types.Bool{Value: true},
-		Upper:      types.Bool{Value: true},
-		Lower:      types.Bool{Value: true},
-		Number:     types.Bool{Value: true},
-		Numeric:    types.Bool{Value: true},
-		MinSpecial: types.Int64{Value: 0},
-		MinUpper:   types.Int64{Value: 0},
-		MinLower:   types.Int64{Value: 0},
-		MinNumeric: types.Int64{Value: 0},
+	state := passwordModelV3{
+		ID:              types.String{Value: "none"},
+		Result:          types.String{Value: id},
+		Length:          types.Int64{Value: int64(len(id))},
+		Special:         types.Bool{Value: true},
+		Upper:           types.Bool{Value: true},
+		Lower:           types.Bool{Value: true},
+		Number:          types.Bool{Value: true},
+		Numeric:         types.Bool{Value: true},
+		MinSpecial:      types.Int64{Value: 0},
+		MinUpper:        types.Int64{Value: 0},
+		MinLower:        types.Int64{Value: 0},
+		MinNumeric:      types.Int64{Value: 0},
+		Keepers:         types.Map{Null: true},
+		OverrideSpecial: types.String{Null: true},
```

Comment:
Do you happen to know what actually gets put into the state if these aren't set? I'm thinking it might be zero-values for the types, so empty map and empty string. 🙁 It'd be a bummer if provider developers explicitly had to set null for Terraform-only attributes on import. I've kind of wanted to support `types` pointers, e.g. `*types.String` in the reflection and potentially drop the `Null` field (and maybe further something like https://github.com/hashicorp/terraform-plugin-framework/issues/441 so you have to explicitly tell the framework that value is known vs unknown), but maybe that is more confusing.

Reply:
If `Keepers` and `OverrideSpecial` are not set in the `ImportState` func, then as you thought, we end up with the zero value for the types in the state. 

```
"keepers": {},
...
"override_special": "",
```

This is problematic as the next _terraform apply_ results in a plan difference.